templates/page/page.tpl.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @file
+ * Returns the HTML for a single Drupal page.
+ *
+ * Complete documentation for this file is available online.
+ * @see https://drupal.org/node/1728148
+ */
+?>
+
+<div id="main" class="main-content" role="main">
+
+  <header class="header" id="header" role="banner">
+
+    <section class="usa-banner">
+      <div class="usa-grid">
+        <div class="usa-banner-content" id="main-content">
+
+          <h1><?php print $site_name; ?></h1>
+
+          <?php if ($site_slogan): ?>
+            <h2 class="usa-font-lead"><?php print $site_slogan; ?></h2>
+          <?php else: ?>
+            <h2 class="usa-font-lead"><?php print t('Open source UI components and visual style
+             guide to create consistency and beautiful user experiences across
+             U.S. federal government websites'); ?></h2>
+          <?php endif; ?>
+
+        </div>
+      </div>
+    </section>
+
+    <section class="usa-section">
+      <div class="usa-grid">
+        <?php print render($page['header']); ?>
+      </div>
+    </section>
+
+  </header>
+
+  <nav>
+
+    <?php print render($page['navigation']); ?>
+
+  </nav>
+
+  <section class="usa-section">
+    <div class="usa-grid">
+      <?php print $breadcrumb; ?>
+      <a id="main-content"></a>
+      <?php print $messages; ?>
+      <?php print render($tabs); ?>
+      <?php print render($page['help']); ?>
+      <?php if ($action_links): ?>
+        <ul class="action-links"><?php print render($action_links); ?></ul>
+      <?php endif; ?>
+      <?php print render($page['content']); ?>
+      <?php print $feed_icons; ?>
+
+      <?php
+        // Render the sidebars to see if there's anything in them.
```

Comment:
I haven't yet spun this branch up locally, but it's possible to just use `if ($page['sidebar_first'])` without rendering first, no? See line 162 of [Bartik](https://www.drupal.org/project/bartik)'s `page.tpl.php` for an example.